src/main/java/org/sagebionetworks/web/client/widget/table/TableListWidget.java

```diff
@@ -73,6 +74,7 @@ public TableListWidget(PreflightController preflightController,
 		this.uploadTableModalWidget = uploadTableModalWidget;
 		this.paginationWidget = paginationWidget;
 		this.createFileViewWizard = createFileViewWizard;
+		this.cookies = cookies;
```

Comment:
Remove this.view.setAddFileViewVisible(DisplayUtils.isInTestWebsite(cookies)); in constructor.